### PR TITLE
Prepare scripts for automated deployment of UserCluster MLA on dev

### PR DESCRIPTION
**Pull request by wurbanski**
**What this PR does / why we need it**:
Implements changes required to automate UserCluster MLA deployments on dev cluster. Part 1 of 2 (TBD: infra PR)

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind feature

**Special notes for your reviewer**:
Needs the accompanying PR in infra (TBD once we settle if this approach is correct)

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

**Comment by wurbanski** [comment]
@xrstf can you take a look at this? I don't want to build entire KKP in this, but I need the installer - is this a good approach here or should I do it differently?

File: hack/ci/deploy.sh [hunk 1]
```diff
@@ -117,6 +117,15 @@ logging)
   deploy "promtail" "logging" charts/logging/promtail/
   ;;
 
+usercluster-mla)
+  echodate "Running Kubermatic Installer for UserCluster MLA..."
+  ./_build/kubermatic-installer deploy usercluster-mla \
+    --config "$KUBERMATIC_CONFIG" \
+    --helm-values "$VALUES_FILE" \
+    --mla-include-iap \
+    --timeout=1500s
```
Comment (xrstf) [inline hack/ci/deploy.sh]: Can we please use a more descriptive timeout, like "1500000000000us"? Or what about simply "25m"?

**Comment by xrstf** [comment]
I don't quite understand the issue here. You want to use the installer, but not build it? And is this PR done or are you still working on it?

File: hack/ci/deploy.sh (hunk 1 above)
Comment (wurbanski) [reply hack/ci/deploy.sh]: it's just `1500000000us` - your proposal would take much too long IMO

**Comment by wurbanski** [comment]
I just didn't know if I should rebuild the installer in the MLA job or is it possible to just use the binary from somewhere else. If I want to rebuild it here before I run it, I will need more resources than if I were only to install. Thought that maybe you would have some idea of how the entire system should run optimally resource-wise, but if it's fine to rebuild it, then we can go with this. 

I used WIP but I meant to just hold it.

/hold

**Comment by xrstf** [comment]
Yeah to be on the safe side you have to rebuild the installer. We generally try to keep Prowjobs self contained and not have dependencies between jobs, plus just compiling the amd64 binaries should be somewhat "quick" (no need to start compiling the arm64 binaries like the user-ssh-keys-agent, which would take ages) :)

**Review by xrstf — APPROVED** [review]
/approve

**Pull request merged**